File: test/fixtures/output/r/httr/full.r
library(httr)

url <- "http://mockbin.com/har"

queryString <- list(
  foo = "bar,baz",
  baz = "abc"
)

payload <- "foo=bar"

encode <- "form"

response <- VERB("POST", url, body = payload, add_headers(cookie = 'foo=bar; bar=baz', accept = 'application/json', , query = queryString, content_type("application/x-www-form-urlencoded"), encode = encode)

content(response, "text")

File: src/targets/ruby/faraday/fixtures/full.rb
require 'faraday'

data = {
  :foo => "bar",
}

conn = Faraday.new(
  url: 'http://mockbin.com',
  headers: {'Content-Type' => 'application/x-www-form-urlencoded'}
)

response = conn.post('/har') do |req|
  req.headers['cookie'] = 'foo=bar; bar=baz'
  req.headers['accept'] = 'application/json'
  req.params['foo'] = ["bar","baz"]
  req.params['baz'] = 'abc'
  req.params['key'] = 'value'
  req.body = URI.encode_www_form(data)
end

puts response.status
puts response.body